### PR TITLE
NL: Start an e2e validator tool

### PR DESCRIPTION
**Pull request by pradh**
Please see included README.

### CODE REVIEW

File: tools/nl/validator/systest.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+  p = cfg['metadata']['placeDcid'][0]
+  if p != pl:
+    ret['WrongPlace'] = '*'
+    counters['WrongPlace'] += 1
+
+  found_sv = False
+  for i, s in enumerate(
+      resp.get('debug', {}).get('sv_matching', {}).get('CosineScore', [])):
+    if s < 0.5:
```
Comment (jehangiramjad) [inline tools/nl/validator/systest.py:169]: why is this needed?

**Review by jehangiramjad — APPROVED** [review]
Thanks. Looks good

File: tools/nl/validator/systest.py (hunk 1 above)
Comment (pradh) [reply tools/nl/validator/systest.py:169]: Since the backend completely ignores such SVs (i.e., they'll never show up on charts), excluding from the rank...those will get INF rank.

**Review by pradh — COMMENTED** [review]
Thanks for the quick review Jehangir!
